packages/theme-wizard-website/e2e/templates.spec.ts

```diff
@@ -20,6 +20,7 @@ test('collage template shows correct content and structure', async ({ collagePag
 
 test('can select different templates from the selector', async ({ themeWizard }) => {
   await themeWizard.selectTemplate('Overzichtspagina');
+  await themeWizard.page.waitForLoadState('domcontentloaded');
```

Comment:
Surprising, the default is to wait for the `load` event, but if it works it works.

Reply:
I'm also a bit surprised, but the trace viewer sometimes showed that the `await expect(templateSelect).toBeVisible();` failed or took a very long, time even though other steps would succeed. I added this also to be a bit more explicit in what is happening. I can still try and remove it, see if it has an impact